device-types/Cisco/FPR4215-NGFW-K9.yaml

```diff
@@ -0,0 +1,47 @@
+---
+manufacturer: Cisco
+model: FPR4215-NGFW-K9
+slug: cisco-fpr4215-ngfw-k9
+part_number: FPR4215-NGFW-K9
+u_height: 1
+is_full_depth: true
+weight: 21
+weight_unit: kg
+airflow: front-to-rear
+comments: https://www.cisco.com/c/en/us/products/collateral/security/firewalls/secure-firewall-4200-ds.html
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: usb
+    type: usb-a
+interfaces:
+  - name: Ethernet1/1
+    type: 10gbase-x-sfpp
```

Comment:
Looking at the spec sheet these are 25GB ports therefore they should be modeled as `25gbase-x-sfp28`